src/components/ProfilePage/ProfilePage.jsx

```diff
@@ -190,8 +190,9 @@ export default function ProfilePage() {
 
                 <div className="user-followed-team-cards">
                     {userTeams[0] ? userTeams.map((team) => { 
-                
+                        console.log(team)
```

Comment:
console log?

Reply:
might be from a previous commit to this branch, but it's removed